website/site/assets/javascripts/lunr/tinyseg.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * export the module via AMD, CommonJS or as a browser global
+ * Export code from https://github.com/umdjs/umd/blob/master/returnExports.js
+ */
+;(function (root, factory) {
+    if (typeof define === 'function' && define.amd) {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Allegheny-Ethical-CS/GatorMiner/pullRequest?prid=7505586)
```suggestion
    if (typeof define === "function" && define.amd) {
```